player/loadfile.c

```diff
@@ -255,24 +257,20 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
         break;
     }
     char b[2048] = {0};
-    bool forced_only = false;
-    if (t->type == STREAM_SUB) {
-        bool forced_opt = mpctx->opts->subs_rend->sub_forced_events_only;
-        if (forced_opt)
-            forced_only = t->selected;
-    }
-    APPEND(b, " %3s %-5s", t->selected ? (forced_only ? "(*)" : "(+)") : "", tname);
-    APPEND(b, " --%s=%d", selopt, t->user_tid);
-    if (t->lang && langopt)
-        APPEND(b, " --%s=%s", langopt, t->lang);
+
+    APPEND(b, "%s %-5s", t->selected ? BLACK_CIRCLE : " ", tname);
```

Comment:
They are aligned for me in Japanese fonts, ● occupies part of the next empty cell but doesn't actually reserve more than 1 cell.

Reply:
I can't figure out this terminal, I tried putting things like `DEFAULT=Source Han Sans JP` in `~/.mlterm/aafont` or making it the default monospace font in `fonts.conf` but it doesn't get used. Anyway the question is if we want to add ○ before all unselected tracks for everyone just for the sake of mlterm configured to use an East Asian font for these circles. I couldn't even get this thing to compile because of incompatible pointer type errors.

Reply:
Still doesn't work, but whatever. Updated to use ○.

Reply:
`○` makes it little bit cluttered. Another question, do we want space at the beginning? I suggested to remove it, because I use `--msg-module`. I'm not sure if space is better without it or not.

Reply:
Well that's why I would have preferred space. Another minor thing is that you can no longer differentiate selected tracks in terminals that don't support UTF-8, but this isn't a big deal in `$current_year`.

I think the space is no longer necessary with the circles visually differentiating tracks from the text around them.